src/data-table/index.js

```diff
@@ -103,7 +103,24 @@ class DataTable extends Component {
   }
 
   columns = () => {
-    const { children, columns } = this.props
+    const { children, columns, data} = this.props
+    const emptyData = []
+    if (!children && !columns) {
+      const columns = []
+      if (data.length > 0) {
+        Object.keys(data[0]).map(key => (key !== '_id' && columns.push({
```

Comment:
* can use `forEach()` instead of `map()` here because we are not returning anything from the function.
* assume we are using forEach here, looks like we can combine this foreach with line 118's map by doing:
```suggestion
        Object.keys(data[0]).map(key => (key !== '_id' && columns.push({
          ...columnDefaultProps,
```
code not tested